ext/cl_khr_subgroups.asciidoc

```diff
@@ -311,6 +329,20 @@ precision is supported) as the type for the arguments.
 
 |====
 
+The *<op>* in *sub_group_reduce_<op>*, *sub_group_scan_inclusive_<op>* and *sub_group_scan_exclusive_<op>* defines the operator and can be *add*, *min* or *max*.
+
+The inclusive scan operation takes a binary operator *op* with an identity I and _n_ (where _n_ is the size of the sub-group) elements [a~0~, a~1~, ... a~n-1~] and returns [a~0~, (a~0~ *op* a~1~), ... (a~0~ *op* a~1~ *op* ... *op* a~n-1~)].
```

Comment:
Great, this was really missing! Do we want to describe the exclusive scan operation as well?

Reply:
Yes, I've added the exclusive scan description as well.

These descriptions were largely taken from the OpenCL C spec descriptions of the work group scan and reduction functions, which include an inclusive scan example.  I think the example is fine but it breaks up the flow of the descriptions.  I'm not going to do anything about this now, but I may try to shuffle some things around for clarity in a subsequent PR.